### ISSUE TITLE
Map-lazy-load error

### ISSUE THREAD

**Issue by alextrs** [allenhwkim/angularjs-google-maps#165]
Hello,

We use ui-router in our site and need map only on one page. I wrapped map directive with map-lazy-load and now first time I visit the page with map everything works as expected, but every consecutive visit of the same page gives me an error:
> You have included the Google Maps API multiple times on this page. This may cause unexpected errors.

This is how I use it:
```javascript
<div class="col-sm-6 col-md-4" ng-if="activeLocation">
   <div map-lazy-load="http://maps.google.com/maps/api/js">
     <map
      ...
    </map>
   </div>
</div>
```

**Comment by allenhwkim** [comment]
I have designed not to load twice if already loaded by checking the script tag. 

Based on your comment, it might be buggy. 
Could you make a plnkr for this, so that I can re-generate your bug?

**Comment by allenhwkim** [comment]
I got it. the script check is not quite right, it will be fixed soon

      if (document.querySelector('script[src="'+src+'?callback=lazyLoadCallback"]')) {
        console.log('foo')
        return false;
      }

**Comment by alextrs** [comment]
Thanks, yep that fixed it

**Comment by allenhwkim** [comment]
fixed with v1.4.3
